### PR TITLE
add file section bio

### PR DESCRIPTION
**Pull request by nesmamanasra**
- add file index.html style.css 
- add section bio
- add div container
- add class
- add image
- add  two paragraphs
- add  css formats
images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,8 +7,24 @@
...
 <body>
-    <div class="main-container" >
+     <section class="section2" >
          
-    </div>
+          <div class="main-container" >
+            <span class="bio" ><p>BIO</p></span>
+            
+                    <div class="content">
+                        <img src="https://static.wixstatic.com/media/84770f_af85ac0d80ee4234a56e871016b9e661~mv2_d_5000_3625_s_4_2.jpg/v1/fill/w_389,h_282,al_c,q_80,usm_0.66_1.00_0.01/84770f_af85ac0d80ee4234a56e871016b9e661~mv2_d_5000_3625_s_4_2.webp">
```
Comment (YakoobHammouri) [inline index.html]: download the image in your project, to use as a local asset

File: index.html [hunk 2]
```diff
@@ -7,8 +7,24 @@
     <title></title>
 </head>
 <body>
-    <div class="main-container" >
+     <section class="section2" >
          
-    </div>
+          <div class="main-container" >
+            <span class="bio" ><p>BIO</p></span>
```
Comment (YakoobHammouri) [inline index.html]: why put the p tag inside the span tag?

File: style.css [hunk 3]
```diff
@@ -47,14 +47,77 @@ h1{
...
-
+ .bio::after{
+    content:'';
+    background: #5386E4;
+    display: block;
+    height: 1px;
+    width: 900px;
+    margin: 40px auto 5px;
+}
+img{
```
Comment (YakoobHammouri) [inline style.css]: recommend using class Name not a tag type, 
by using the img tag all image in the site will take this CSS

File: style.css [hunk 4]
```diff
@@ -47,14 +47,77 @@ h1{
...
+    font-size: 14px;
+    line-height: 2em;
+    vertical-align: baseline;
+   font-family: 'palatino linotype',serif;
+    color: #3D3D3D;
+    pointer-events: auto;
+    text-align: start;
+   
+}
+.section2{
```
Comment (YakoobHammouri) [inline style.css]: the name must be descriptive

File: style.css [hunk 5]
```diff
@@ -47,14 +47,77 @@ h1{
...
+    top:0;
+    bottom: 0;
+    width: 100%;
+     position: relative;
+     height: 580px;
+    background-color:#f6f6f6;
+}
+.content{
+    width:900px;
```
Comment (YakoobHammouri) [inline style.css]: why using fixed width?

File: style.css [hunk 6]
```diff
@@ -47,14 +47,77 @@ h1{
...
+.bio{
+    margin-top: 30px;
+    margin-bottom:15px ;
+    color: #5386E4;
+    font:normal normal normal 30px/1.4em spinnaker,sans-serif;
+    font-size: 40px;
+    text-align: center;
+    letter-spacing: 0.2em;
+}
+.str{
```
Comment (YakoobHammouri) [inline style.css]: the name must be descriptive

File: style.css [hunk 7]
```diff
@@ -47,14 +47,77 @@ h1{
...
+    width: 900px;
+    margin: 40px auto 5px;
+}
+img{
+    float: left;
+    margin-top: 50px;
+    
+    
+}
+.par1{
```
Comment (YakoobHammouri) [inline style.css]: what is the name convention using her?

File: style.css [hunk 8]
```diff
@@ -47,14 +47,77 @@ h1{
...
+    letter-spacing: 0.08em;
+    font-size: 14px;
+    line-height: 2em;
+    vertical-align: baseline;
+   font-family: 'palatino linotype',serif;
+    color: #3D3D3D;
+    pointer-events: auto;
+    text-align: start;
+}
+.par2{
```
Comment (YakoobHammouri) [inline style.css]: what is the name convention using her?

**Review by YakoobHammouri — CHANGES_REQUESTED** [review]

File: style.css [hunk 9]
```diff
@@ -47,14 +47,77 @@ h1{
...
+img{
+    float: left;
+    margin-top: 50px;
+    
+    
+}
+.par1{
+    width:224px;
+    height: 300px;
+    float: left;
```
Comment (YakoobHammouri) [inline style.css]: do not use float, use flex

File: style.css [hunk 10]
```diff
@@ -47,14 +47,77 @@ h1{
...
+    vertical-align: baseline;
+   font-family: 'palatino linotype',serif;
+    color: #3D3D3D;
+    pointer-events: auto;
+    text-align: start;
+}
+.par2{
+    width:224px;
+    height: 300px;
+    float: left;
```
Comment (YakoobHammouri) [inline style.css]: do not use float, use flex

File: style.css [hunk 11]
```diff
@@ -47,14 +47,77 @@ h1{
...
+    line-height: 2em;
+    vertical-align: baseline;
+   font-family: 'palatino linotype',serif;
+    color: #3D3D3D;
+    pointer-events: auto;
+    text-align: start;
+}
+.par2{
+    width:224px;
+    height: 300px;
```
Comment (YakoobHammouri) [inline style.css]: do not use fix width

File: style.css [hunk 12]
```diff
@@ -47,14 +47,77 @@ h1{
...
+}
+.section2{
+    top:0;
+    bottom: 0;
+    width: 100%;
+     position: relative;
+     height: 580px;
+    background-color:#f6f6f6;
+}
+.content{
```
Comment (YakoobHammouri) [inline style.css]: change  to flex

File: style.css [hunk 13]
```diff
@@ -47,14 +47,77 @@ h1{
...
+}
+img{
+    float: left;
+    margin-top: 50px;
+    
+    
+}
+.par1{
+    width:224px;
+    height: 300px;
```
Comment (YakoobHammouri) [inline style.css]: do not use fix width

**Review by YakoobHammouri — CHANGES_REQUESTED** [review]

**Review by YakoobHammouri — APPROVED** [review]

File: index.html [hunk 14]
```diff
@@ -7,8 +7,26 @@
...
 </head>
 <body>
-    <div class="main-container" >
+     <section class="section-bio" >
          
-    </div>
+          <div class="main-container" >
+            <div class="bio-header"><p>BIO</p></div>
+            
+                    <div class="bio-box ">
```
Comment (sarah-alsharif0) [inline index.html]: I recommend using the BEM convention in naming the classes like : 
`bio__header` , `bio__box`
like what the mentor @YakoobHammouri has said.

**Review by sarah-alsharif0 — CHANGES_REQUESTED** [review]

**Review by MontaserSaleh91 — APPROVED** [review]

**Pull request merged**